/usr/ports/pobj/tdesktop-6.2.5/tdesktop-6.2.5-full/Telegram/SourceFiles/platform/linux/specific_linux.cpp:71:7: error: reference to 'GObject' is ambiguous

plus fix scroll on X11
https://github.com/desktop-app/lib_ui/pull/339

Index: Telegram/SourceFiles/platform/linux/specific_linux.cpp
--- Telegram/SourceFiles/platform/linux/specific_linux.cpp.orig
+++ Telegram/SourceFiles/platform/linux/specific_linux.cpp
@@ -66,7 +66,7 @@ void PortalAutostart(bool enabled, Fn<void(bool)> done
 		Gio::DBusProxyFlags::NONE_,
 		base::Platform::XDP::kService,
 		base::Platform::XDP::kObjectPath,
-		[=](GObject::Object, Gio::AsyncResult res) {
+		[=](auto, Gio::AsyncResult res) {
 			auto proxy = XdpBackground::BackgroundProxy::new_for_bus_finish(
 				res);
 
@@ -119,7 +119,7 @@ void PortalAutostart(bool enabled, Fn<void(bool)> done
 					+ '/'
 					+ handleToken,
 				nullptr,
-				[=](GObject::Object, Gio::AsyncResult res) mutable {
+				[=](auto, Gio::AsyncResult res) mutable {
 					auto requestProxy = XdpRequest::RequestProxy::new_finish(
 						res);
 
@@ -194,7 +194,7 @@ void PortalAutostart(bool enabled, Fn<void(bool)> done
 								GLib::Variant::new_variant(
 									GLib::Variant::new_boolean(false))),
 						}),
-						[=](GObject::Object, Gio::AsyncResult res) mutable {
+						[=](auto, Gio::AsyncResult res) mutable {
 							auto &sandbox = Core::Sandbox::Instance();
 							sandbox.customEnterFromEventLoop([&] {
 								const auto result =
@@ -449,7 +449,7 @@ bool GenerateServiceFile(bool silent = false) {
 		Gio::DBusProxyFlags::NONE_,
 		base::Platform::DBus::kService,
 		base::Platform::DBus::kObjectPath,
-		[=](GObject::Object, Gio::AsyncResult res) {
+		[=](auto, Gio::AsyncResult res) {
 			auto interface = XdgDBus::DBus(
 				XdgDBus::DBusProxy::new_for_bus_finish(res, nullptr));
 
@@ -550,7 +550,7 @@ void PortalCheckScheme(
 		Gio::DBusProxyFlags::NONE_,
 		base::Platform::XDP::kService,
 		base::Platform::XDP::kObjectPath,
-		[=](GObject::Object, Gio::AsyncResult res) {
+		[=](auto, Gio::AsyncResult res) {
 			auto interface = XdpOpenURI::OpenURI(
 				XdpOpenURI::OpenURIProxy::new_for_bus_finish(res, nullptr));
 
@@ -564,7 +564,7 @@ void PortalCheckScheme(
 				GLib::Variant::new_array(
 					GLib::VariantType::new_("{sv}"),
 					{}),
-				[=](GObject::Object, Gio::AsyncResult res) mutable {
+				[=](auto, Gio::AsyncResult res) mutable {
 					const auto result
 						= interface.call_scheme_supported_finish(res);
 
